engine/table/python/core/deephaven_jpy_init.py

```diff
@@ -18,188 +16,5 @@
-    return java_array('int', values)
-
-
-def DoubleArray(values):
-    return java_array('double', values)
-
-
-def FloatArray(values):
-    return java_array('float', values)
-
-
-def LongArray(values):
-    return java_array('long', values)
-
-
-def ShortArray(values):
-    return java_array('short', values)
-
-
-def BooleanArray(values):
-    return java_array('boolean', values)
-
-
-def ByteArray(values):
-    return java_array('byte', values)
-
-
-def _exists_and_is_file_(prop):
-    """
-    For finding a fully qualified customer configured import file
-
-    :param prop: a stem of the import file
-    :return: the fully qualified file path
-    """
-
-    config = Config()
-    file_path = config.getStringWithDefault(prop, None)
-    if file_path:
-        file_path = file_path.replace("<devroot>", config.getDevRootPath())
-        if os.path.isfile(file_path):
-            return file_path
-
-    return None
-
-
-def _empty_or_comment_(line):
-    """helper method for extracting a line"""
-    return line is None or len(line.strip()) < 1 or line.strip().startswith("#")
-
-
-######################################################################################################
-# Perform the desired imports for the console workspace
-# NOTE: these can't be moved into a method or the imports would be out of scope for the console
-
-default_imports = _exists_and_is_file_("python.default.imports")
-if default_imports:
-    # test if line matches pattern "import <stuff>" or "from <stuff> import <stuff>"
-    import re
-
-    import_re = '^(?:import|(from(\s+)(.+))import)(\s+)(.+)'
-    import_pattern = re.compile(import_re)
-
-    with open(default_imports, 'r') as f:
-        for line in f:
-            # note that this pattern is repeated, but I want to avoid a method polluting the namespace
-            if _empty_or_comment_(line):
-                # this line is empty or a comment
-                continue
-
-            if ";" in line:
-                logging.error("Can not run line \n{}\n from python.default.imports, contains a ';'".format(line))
-                continue
-
-            try:
-                if import_pattern.match(line):
-                    exec(line)
-                else:
-                    logging.error("Could not run line \n{}\n from python.default.imports, "
-                                  "does not match import pattern" .format(line))
-            except ImportError:
-                logging.error("Could not import module: {}".format(line))
-    del import_re, import_pattern
-
-default_imports = _exists_and_is_file_("python.default.javaclass.imports")
-if default_imports:
-    with open(default_imports) as f:
-        for line in f:
-            if _empty_or_comment_():
-                continue
-
-            # noinspection PyBroadException
-            try:
-                importjava(line)
-            except Exception:
-                logging.error("Could not import java class: {}".format(line))
```

Comment:
This section comes back to importing java methods into the query language.  DHE has the ability to preconfigure user-specified functions to be in the query language.  It is a reasonable feature.

How best to handle this and the imports is an interesting question.  Simply supporting a (1) preexec script and (2) a `QueryLibrary` wrapper, you could cover both of the use cases here, with less specific code.